core/src/main/java/org/elasticsearch/node/InternalSettingsPreparer.java

```diff
@@ -44,6 +47,9 @@
 
 public class InternalSettingsPreparer {
 
+    private static Logger logger = ESLoggerFactory.getLogger("settings");
```

Comment:
I was trying to make the context in the log message clearer (that it had to do with settings) vs a user thinking "what is InternalSettingsPreparer". But I changed it as you suggested. If we don't want to allow using `getLogger(String)`, we should remove it.